Makefile

```diff
@@ -9,6 +9,7 @@ PKGBASE=github.com/lf-edge/eve
 GOMODULE=$(PKGBASE)/pkg/pillar
 GOTREE=$(CURDIR)/pkg/pillar
 PATH:=$(CURDIR)/build-tools/bin:$(PATH)
+ROOTFS_SIZE_MB:=350
```

Comment:
As discussed -- lets drop these changes for now. We don't need to include SPDK in the default image anyway.